theme/themes/pxt/globals/site.variables

```diff
@@ -33,13 +34,13 @@
      Fonts
 --------------------*/
 
-@importGoogleFonts: false;
+@importGoogleFonts: true;
 
-@docsHeaderFont: "Helvetica Neue", Helvetica, Arial, sans-serif;
-@docsPageFont: "Helvetica Neue", Helvetica, Arial, sans-serif;
+@docsHeaderFont: Lato,"Helvetica Neue", Helvetica, Arial, sans-serif;
```

Comment:
Where is that font coming from? We probably need to file an oss request for it.

Reply:
Yes, this is the new "Lato" font we were using from google font. We could donwload the web font and store it in docs/static/fonts, is that ok? That would solve us the issue of getting the font from google

Reply:
What is the license for that font? We need to file an OSS request for it.

Reply:
Does it support non-European scripts?

Reply:
Let me look into adding to pxt myself. There is some administrative task attached to this. Maybe remove it from this PR and make it a separate PR?

Reply:
Let me look into adding to pxt myself. There is some administrative task attached to this. Maybe remove it from this PR and make it a separate PR?

Reply:
Right we probably need a fallback font.